content/200-concepts/100-components/01-prisma-schema/07-referential-actions.mdx

```diff
@@ -0,0 +1,327 @@
+---
+title: 'Referential actions'
+metaTitle: 'Referential actions'
```

Comment:
Other Preview features have `(Preview)` in their page title, see https://deploy-preview-1920--prisma2-docs.netlify.app/docs/concepts/components/preview-features/sql-server

Reply:
You mean like this `metaTitle: 'Microsoft SQL Server Connector (Preview)'`

Reply:
Yes, exactly. 
As long as the functionality is behind a preview feature flag, that needs toe clearly be communicated which usually includes the title in navigation and page title.